app/models/profile.rb

```diff
@@ -42,14 +42,16 @@ class Profile < ApplicationRecord
   belongs_to :account
   delegate :s3_provider_prefix, to: :account
 
+  LOGO_STYLES = ->(logo) { {large: '300x300>'.freeze, medium: '150x150>'.freeze, thumb: '100x100>'.freeze, invoice: (logo.content_type.end_with?("gif") ? ['200x50>'.freeze, :png].freeze : '200x50>'.freeze)}.freeze }
```

Comment:
For the sake of readability and style, maybe make this multi-line?
Also, maybe remove `.freeze` and add `# frozen_string_literal: true`?
Finally, maybe use single quotes for strings.

Reply:
I don't want to change whole profile. We have a number of files without it. To reduce scope of possible issues.

Reply:
made multi-line

which single quote do you mean?

Reply:
I meant more like:
```
  LOGO_STYLES = ->(logo) {
    {
      large: '300x300>'.freeze,
      medium: '150x150>'.freeze,
      thumb: '100x100>'.freeze,
      invoice: (logo.content_type.end_with?('gif') ? ['200x50>'.freeze, :png].freeze : '200x50>'.freeze)
    }.freeze
  }
```

Just easier to read what styles are available and what are their sizes.
Re: quotes I meant `"gif"` => `'gif'`, for consistency.

Reply:
makes sense, thanks!